### ISSUE TITLE
Want to erase a super large text as Response body

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/BrowserMobProxyInKatalonStudio#22]
https://github.com/kazurayam/BrowserMobProxyInKatalonStudio/releases/tag/1.5.0
supported transforming a large HAR into a smaller one.
I could select the entries that I am interested in and drop the rest off.
For example I could select the entries of which Request is targeted to URL that matches certain regex, such as `.*jquery\.min\.js`.
This was a great step forward.

However, I was not contented with the v1.5.0 achievement.

A quote from the README of v1.5.0

>When the script is executed, you will find a file with name *-selection.har will be created. Please find that the size of *-selection.har is much smaller than the *-source.har file.
```
$ ls -la work | grep TS1
-rw-r--r--   1 kazuakiurayama  staff     959232  7 24 17:00 TS1_demoaut.katalon.com-pretty.har
-rw-r--r--   1 kazuakiurayama  staff     905344  7 24 17:00 TS1_demoaut.katalon.com-source.har
-rw-r--r--   1 kazuakiurayama  staff     154419  7 24 17:00 TS1_demoaut.katalon.com-selection.har
```

The `-selection.har` file is as large as 1.5MB. Why? I selected only 2 entries, so I expected the `-selection.har` could be small. As small as 100KB.

I found the reason. The entry of URK that matchs `.*jquery.min.js` has a content text of which size is huge.

```
"request": {
          "method": "GET",
          "url": "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.min.js",
          "httpVersion": "HTTP/1.1",
          ...
        },
        "response": {
          "status": 200,
          "statusText": "OK",
...
          "content": {
            "size": 29929,
            "mimeType": "application/javascript; charset=utf-8",
            "text": "W/d2EUWwcQBAvP+ENSJZ7SUqoJcDze5zahusrCZV9IzlcK71qXU69MtpSlxh8gnGe2bpbnKtwauPuZjfp2Ro8nyElj6S5O//96p9/UpjnesZGISz+x55fPNw02Ybx27ttPbh4hwkkEwDSAv4PYpZqdnfnx/gNapppoGdtccLKZEOmECKkqho57nzqzStv8/lNFWr2AU+ZQBSmrUMhTEeajHyZWZZwHc3u4ngh+//11TZnKj3F8jfECiUPJO2rl1VhihtZRTvBF36krf+dAaUQyBYOnjENph2xMNmBbpkhIPHF5ipVqtaLtIFzWoWvL+wkyz8TWX4+yv5DBGtqdL/pSsH9NiyrYq6xTDB92pWZbk7hQE3o/O1OpwVnN/EOZ5Mo0TLk4IM6D81qyx3aw8EXuMJHeuSk/x7Ow4aBlhSRWsOcaZUcy74/5epvdtqEBQ0DjGtfPaJoOSUd2mzV9/w7kelFququwcdgFE3QAwBkrIAkrKp9EKBqq7GHFd3Qz4NEPYBQX0r+hyGH/RDpGb088o5bLbSOOfN0luv7L913BrKiSjpJMd2mmaVXieAwfZF1e+ldeyuJHcS1wRAEKSUfw1t/
...(a long character string continue)
```

I do not want the character string longer than 200 characters as `text` in the response. I want to forcibly replace the string to a shorter string, like "SHORTENED".

**Comment by kazurayam** [comment]
XSLT supports this sort of transformation for XML documents.
I want to do the similar transformation for JSON.

**Comment by kazurayam** [comment]
Other tools may support it.
Let me look at JMESPath.

**Comment by kazurayam** [comment]
It looks that JMESPath has something 

https://stackoverflow.com/questions/68538114/jmespath-conditional-string-replacement

**Comment by kazurayam** [comment]
Possibly I want to apply the [Java Visitor Pattern](https://www.baeldung.com/java-visitor-pattern) on the object.

**Comment by kazurayam** [comment]
https://github.com/ColinEberhardt/json-transforms
 looks interesting

**Comment by kazurayam** [comment]
I have worked on various JSON transformation libraries for a few days.

And I have found out a simple solution.

**Comment by kazurayam** [comment]
I released the new v2.1.0 

https://github.com/kazurayam/BrowserMobProxyInKatalonStudio/blob/2.1.0/Scripts/main/post-process/HAR%20Transform/Script1721511804476.groovy

has a few lines added

```

```

**Comment by kazurayam** [comment]
done at v2.1.0
